test/e2e/tests/test_endpoint.py

```diff
@@ -33,8 +33,8 @@
 from e2e.replacement_values import REPLACEMENT_VALUES
 from e2e.common import config as cfg
 
-FAIL_UPDATE_ERROR_MESSAGE = "unable to update endpoint. check FailureReason"
-
+FAIL_UPDATE_ERROR_MESSAGE = "EndpointUpdateError: unable to update endpoint. check FailureReason. latest EndpointConfigName is "
+LAST_ENDPOINTCONFIGNNAME_FOR_UPDATE_ANNOTATION_KEY = "sagemaker.services.k8s.aws/Endpoint-last-endpoint-config-for-update"
```

Comment:
nit: lets keep the variable name length small  
you can write a comment above it to explain exactly what it is used for if necessary 

should be fine for now